test/Serilog.Sinks.MSSqlServer.Tests/Sinks/MSSqlServer/Output/StandardColumnDataGeneratorTests.cs

```diff
@@ -63,6 +63,30 @@ public void GetStandardColumnNameAndValueForMessageReturnsSimpleTextMessageKeyVa
             Assert.Equal(messageText, result.Value);
         }
 
```

Comment:
Please also add a test with DataLength = -1

Reply:
Extra test added, and renamed some previously added tests for consistency.